main.py

```diff
@@ -18,29 +18,29 @@ def __init__(self, defaultlang):
     def _(self, text):
         return self.ugettext(text)
 
-    def fbind(self, name, func, args, **kwargs):
+    def fbind(self, name, func, *args, **kwargs):
         if name == "_":
             self.observers.append((func, args, kwargs))
         else:
-            return super(Lang, self).fbind(name, func, *largs, **kwargs)
+            super(Lang, self).fbind(name, func, *args, **kwargs)
```

Comment:
Just noticed in the kivy documentation (https://kivy.org/doc/stable/api-kivy.event.html) that fbind should actually return an uid. For to accomplish this how would the uid be created for the case where name == '_'?